app/uk/gov/hmrc/plasticpackagingtaxregistrationfrontend/views/error_template.scala.html

```diff
@@ -1,3 +1,19 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
 @import uk.gov.hmrc.plasticpackagingtaxregistrationfrontend.config.AppConfig
 
 @this(govuk_wrapper: govuk_wrapper)
```

Comment:
AppConfig can be injected here, or just in the `govuk_wrapper`.

Right now you are injecting this into Controller pass to view pass to govuk_wrapper.

If you inject this directly in govuk wrapper you can forgot about first two steps